docs/changes/io.fits/14528.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+``ImageHDU`` now properly rejects zero-dimensional data, avoiding data
+corruption.
```

Comment:
Maybe "rejects Numpy scalars" to be more precise ? Python scalars are already rejected.